src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingResponseMapper.java

```diff
@@ -48,37 +49,40 @@ public GetHearingResponse toHearingResponse(HearingEntity hearingEntity) {
 
     private RequestDetails setRequestDetails(HearingEntity hearingEntity) {
         RequestDetails requestDetails = new RequestDetails();
+        CaseHearingRequestEntity caseHearingRequestEntity = hearingEntity.getLatestCaseHearingRequest();
```

Comment:
Changes in this class around now getting the latest hearing request - prior to auditing changes there was only one ever available.